tools/analysis_tools/sot/sot_plot_curve.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import os.path as osp
+
+import mmengine
+import numpy as np
+
+from mmtrack.utils import (plot_norm_precision_curve, plot_precision_curve,
+                           plot_success_curve)
+
+
+def main():
+    parser = argparse.ArgumentParser(description='sot plot')
+    parser.add_argument(
+        'sot_eval_res',
+        help='the json/yaml/pickle file path of evaluation results. The '
+        'content of the file must be a collection of name/value pairs. The '
+        'name is a tracker name. The value is also a collection of name/value '
+        'pairs in the format dict(success=np.ndarray, '
+        'norm_precision=np.ndarray, precision=np.ndarray)')
```

Comment:
the saved file from `sot_metrics` should be consistent with the input of `sot_eval_res`